Fibo1.js

```diff
@@ -10,4 +10,4 @@ function fibonaci(n) {
     }
 }
 
-console.log('num is', fibonaci(20));
\ No newline at end of file
+console.log('num is loacker', fibonaci(20));
```

Comment:
Please add logstash integration, I can't push this PR with the current basic logging.